src/status_im/contexts/wallet/router/schema.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im.contexts.wallet.router.schema)
+
+(def ?fee-modes
+  [:enum
+   :tx-fee-mode/custom
+   :tx-fee-mode/normal
+   :tx-fee-mode/fast
+   :tx-fee-mode/urgent])
+
+(def ^:private ?hex
+  [:and :string [:re #"^0x[0-9a-fA-F]+$"]])
+
+(def ^:private ?suggested-levels-for-max-fees-per-gas
+  [:map {:closed? true}
+   [:low ?hex]
+   [:medium ?hex]
+   [:high ?hex]
+   [:low-estimated-time :int]
+   [:medium-estimated-time :int]
+   [:high-estimated-time :int]
+   [:low-priority ?hex]
+   [:medium-priority ?hex]
+   [:high-priority ?hex]])
```

Comment:
`?hex` could live in the common namespace and be used as `:schema.common/hex` as this can promote reusability of foundational types.